src/briefcase/exceptions.py

```diff
@@ -90,9 +90,12 @@ def __str__(self):
 
 
 class NetworkFailure(BriefcaseCommandError):
-    def __init__(self, action):
+    DEFAULT_HINT = "is your computer offline?"
```

Comment:
The default hint is meant to prevent needing to modify other code paths that already relied on `NetworkFailure`. Doing the change this way prevents those areas from needing to change or duplicate the generic hint.